src/util/Enums.js

```diff
@@ -61,4 +61,8 @@ export default {
 
   //IDP Discovery
   IDP_DISCOVERY: 'IDP_DISCOVERY',
+
+  //App Store Links
+  OKTA_VERIFY_APPLE_APP_STORE_URL: 'https://apps.apple.com/us/app/okta-verify/id490179405',
```

Comment:
is the id number in the link is always constant ?

Reply:
Yes, it's an id of an app in app store. Should be a constant.
cc @yannongli-okta